docs/time/index.rst

```diff
@@ -129,7 +129,7 @@ can only have scales in which one day is always equal to 86400 seconds.
    '2007-01-01T04:21:49.579' '2008-01-01T10:54:33.386'
    '2008-12-31T17:27:17.193' '2010-01-01T00:00:00.000']>
 
-  >>> t.sidereal_time('apparent', 'greenwich')  # doctest: +FLOAT_CMP
+  >>> t.sidereal_time('apparent', 'greenwich')  # doctest: +FLOAT_CMP +REMOTE_DATA
```

Comment:
I thought about setting `iers.conf.auto_download = False` in the doc here too, but decided against it. I think it is cleaner this way unless you strongly oppose this approach.

Reply:
Agreed; docs should be kept clean.